### PR TITLE
Fix inertia middleware after v0.3 update

### PR DESCRIPTION
**Pull request by WindOfRussia**
## Summary
Restore inertia middleware for handling component autoreloads after ajax calls. The recent version bump of `inertiajs/inertia-laravel` broken this functionality.

Additional PR changes include: 
- removing unnecessary permissions route
- adding `security-advisories` as a composer firewall against risky packages.

### Current Behavior
When you visit roles. Create a role with no name. The error is shown but the create button remains greyed out until a full page reload.

### Intended Behavior
When the form is posted and returns a response, inertia should reset the processing state of the form.

### CODE REVIEW

**Review by Alex-Hayeur — APPROVED** [review]
Looks good, were the functions in HandleInteriaRequests generated?

**Comment by WindOfRussia** [comment]
> Looks good, were the functions in HandleInteriaRequests generated?

Yeah. They changed the way share / version works.

File: composer.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
         "spatie/laravel-permission": "^3.17"
     },
     "require-dev": {
+        "roave/security-advisories": "dev-master",
```
Comment (EvanDime) [inline composer.json:23]: What is this?

**Review by EvanDime — APPROVED** [review]
Everything is in order here. Though the middleware seems a bit too basic, from what I can tell.

File: composer.json (hunk 1 above)
Comment (alexstojda) [reply composer.json:23]: Ditto, could you explain this library @WindOfRussia

**Review by alexstojda — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (WindOfRussia) [reply composer.json:23]: Screen Shot 2020-10-16 at 2 36 34 PM

Think of it as a firewall so we don't install bad packages

more info: https://github.com/Roave/SecurityAdvisories

**Review by alexstojda — APPROVED** [review]

**Review by EvanDime — APPROVED** [review]

**Pull request merged**